### DISCUSSION TITLE
Is the v1 docs also good for v2?

### DISCUSSION THREAD

**Discussion by GF-Huang** [Live-Charts/LiveCharts2#217] [Q&A]
https://lvcharts.net/App/examples/Wpf/start

**Comment by beto-rodriguez** [comment]
No, v2 is based on v0, but the syntax is not identical.

**Reply by GF-Huang** [reply]
Is it good to use v2 to starting new project at this time even if docs and samples is incomplete?

**Reply by beto-rodriguez** [reply]
There are many new features and improvements compared with v0, but there are some missing features such as PointStates.

**Comment by CodeWithMichal** [comment]
No, dont rely on v1 docs. There are many similarities but its not the same and a lot new things.
Docs and samples are there:
https://github.com/beto-rodriguez/LiveCharts2/tree/master/docs/overview
https://github.com/beto-rodriguez/LiveCharts2/tree/master/samples

**Reply by jean309** [reply]
Hi, Thanks for this work.
We already use lvchart but would like to see what's new in version 2 in a simple way
Do you plan to update https://lvcharts.net/App/examples/Wpf/start?

**Reply by beto-rodriguez** [reply]
Hi @jean309 yes, the site will be updated soon, the idea is that version 2 supports all the features v0 did, but it now works in dotnet 5, 6 and new platforms such as Blazor-wasm, Maui and more.
